Resources/public/css/views/notificationhub.css

```diff
@@ -11,5 +11,5 @@
 .is-navigationhubview-fixed .ez-view-notificationhubview,
 .is-navigation-hidden .ez-view-notificationhubview {
     width: 100%;
-    position: fixed;
+    position: relative;
```

Comment:
I have removed position attribute at all. `.ez-view-notificationhubview` is inside `.ez-notification-container` which is positioning with fixed so here is no reason to keep it.